module3/projects/take_home/rubric.md

```diff
@@ -0,0 +1,160 @@
+- is unable to translate their thoughts into code. 
+- is unable to speak to code or describe what it does. 
+- does not seek to break out their code into reusable components, and doesn’t understand the value of doing so.
+</section>
+
+<section class="dropdown">
+### 3. Testing and Debugging
+#### Strong Yes Hire
+
+The candidate...
+- tests all sad paths and edge cases. 
+- if any unexpected bugs appear during the interview, the candidate effectively debugs the issue.
+
+#### Leaning Yes Hire
+
+The candidate...
+- tests all sad paths but may have missed some edge cases. 
+- tests all happy paths and feature work. 
+- begins a debugging process and posits some reasonable solutions to the problem if any unexpected bugs appear during the interview.
+
+#### Leaning Don’t Hire
+
+The candidate...
+- tests happy paths but missed, or inffecetively tested, some sad paths. 
+- could not identify edge cases. 
+- begins a basic debugging process but is not able to come up with any solutions to the problem if any unexpected bugs appear during the interview.
+
+#### Strong Don’t Hire
+
+The candidate...
+- does little to no testing. 
+- does not spot glaring bugs in the code, if any, or does not attempt to debug when found.
+</section>
+
+<section class="dropdown">
+### 4. Presentation and Communication
+#### Strong Yes Hire
+
+The candidate...
+- frequently checks understanding when explaining a complex concept, and if there is any misunderstanding, is able to unpack the problem and effectively communicate the necessary information. 
+- finds the precise word they want to convey the exact meaning they need (even if they take a few seconds to do so). 
+- is able to explain with great precision why they took the steps they did on a project. 
+- has put together cases that persuade a skeptical audience.
+
+#### Leaning Yes Hire
+
+The candidate...
+- describes how their code works, unprompted. 
+- explains their thought processes very clearly using proper technical terms. 
+- clearly imparts knowledge and concepts through appropriate means (ex: diagrams, body movement, writing and speech). 
+- talks at a level appropriate to their audience When explaining something. 
+
+#### Leaning Don’t Hire
+
+The candidate...
+- needs to say things multiple times to get their point across to a good faith listener. 
+- can communicate an idea at a high level, but lacks precision and nuance in the details. 
+- “handwaves” complex parts of code despite being prodded to expound. 
+
+#### Strong Don’t Hire
+
+The candidate...
+- intent is frequently unclear. 
+- cannot clearly communicate their intentions or thought process. 
+- spent little to no time explaining their thought processes. 
+- Interviewer had to actively prompt the candidate with questions.
+
+</section>
+
+<section class="dropdown">
+### 5. Documentation and Planning
+#### Strong Yes Hire
+
+The candidate...
+- includes a README with a walkthrough of the FE or Swagger for the BE to run requests. 
```

Comment:
"a walkthrough of the FE" do you mean like a screen recording? A gif or a video with audio? I think we'll want to clarify.